Gemfile.lock

```diff
@@ -149,6 +149,10 @@ GEM
       activesupport (= 7.0.4.3)
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225376267-daca2d73-ed71-4974-8f12-d1b72abad205.png)

- [ ] It would be great if you add the **Install** section after Setup and before the Usage section to make your readme professional 👍 
- [ ] Kindly add **Run tests** section after Usage section to make your readme file useful for the new develpers or viewers of your application 👍

## Setup
```
cd my-folder
git clone git@github.com:enoqueJonas/blog-app-ror.git
cd blog-app-ror
```
## Install
```
bundle install
rails db:create
rails db:migrate
```
## Usage
```
rails server or rails s
```
## Run tests
```
rspec spec
```